File: Java/CodingBat/luckySum.java
// http://codingbat.com/prob/p130788

public int luckySum(int a, int b, int c) {
  int[] elements = new int[] { a, b, c };
  int sum = 0;
  
  for (int i = 0; i < elements.length; ++i) {
    if (elements[i] == 13) {
      return sum;
    }
    else {
      sum += elements[i];
    }
  }
  
  return sum;
}

File: Python/CodingBat/lucky_sum.py
# http://codingbat.com/prob/p107863

def lucky_sum(a, b, c):
  numbers = [a, b, c]
  lucky_sum = 0
  
  for number in numbers:
    if number == 13:
      break
    else:
      lucky_sum += number
      
  return lucky_sum
